test/functional/wallet_bumpfee.py

```diff
@@ -253,18 +236,6 @@ def test_small_output_with_feerate_succeeds(self, rbf_node, dest_address):
     rbf_node.generatetoaddress(1, rbf_node.getnewaddress())
     assert_equal(rbf_node.gettransaction(rbfid)["confirmations"], 1)
 
-def test_dust_to_fee(self, rbf_node, dest_address):
```

Comment:
might want to think about how to convert this test to feerate. It seems like a worthwhile testing gap currently.

Reply:
thanks! -- done